shards/shards.go

```diff
@@ -1064,7 +1065,11 @@ func (s *shardedSearcher) getLoaded() loaded {
 
 func mkRankedShard(s zoekt.Searcher) *rankedShard {
 	q := query.Const{Value: true}
-	result, err := s.List(context.Background(), &q, nil)
+	ctx, err := systemtenant.With(context.Background())
+	if err != nil {
+		return &rankedShard{Searcher: s}
+	}
```

Comment:
given you are passing in a context.Background, should we just export the systemtenant context to avoid this extra bit of error handling?

Reply:
I wanted to force people to give us a context so that we can check it has not already been tainted. Exporting a systemtenant feels too easy to mess up? Not feeling to strong about this, though